app/talk/discussion-preview.cjsx

```diff
@@ -62,8 +62,9 @@ module.exports = React.createClass
       <div className="preview-content">
 
         {if @state.subject?
+          subject = getSubjectLocation(@state.subject)
           <div className="subject-preview">
-            <Thumbnail src={getSubjectLocation(@state.subject).src} width={100} />
+            <Thumbnail src={subject.src} format={subject.format} width={100} controls={false} />
```

Comment:
That's fine for people who use a mouse, or similar pointing device, to interact with the video, but renders it inaccessible for people who primarily interact with the page via the keyboard. I'm not sure how big of a deal that is for the small previews, but it's something to bear in mind when adding click interactions to DOM elements that aren't buttons or links.

Do you think it would be better if these small previews were wrapped in links, so that clicking on them went through to the subject page? There might actually be a very old PFE issue asking for this change for preview images.